src/js/player.js

```diff
@@ -2931,6 +2931,45 @@ class Player extends Component {
     if (isEvented(this)) {
       this.trigger('playerreset');
     }
+    this.resetControlBarUI();
+  }
+
+  /**
+   * Reset Control Bar's UI by calling sub-methods that reset
+   * all of Control Bar's components
+   */
+  resetControlBarUI() {
```

Comment:
these methods should be suffixed with an `_` to show that they're "private" or shouldn't be used directly outside of Video.js.